proto/spacemesh/v1/smesher_types.proto

```diff
@@ -129,3 +129,14 @@ message PostStatusResponse {
 message PostDataCreationProgressStreamResponse {
     PostStatus status = 1;
 }
+
+message EstimatedRewardsRequest {}
+
+message EstimatedRewardsResponse {
+    // The amount of the total estimated reward in the next upcoming epoch
+    Amount amount = 1;
+    // Epoch duration
+    SimpleInt duration = 2;
```

Comment:
I think this is field is redundant because the epoch duration in time units is part of a network config data that all clients already have.

Reply:
`data_size` should be part of the config, too, but it isn't currently exposed by any API endpoints so I'm going to leave it here for now